README.md

```diff
@@ -65,6 +65,18 @@ Github](http://github.com/belluzj/cosmic-sans-neue/issues) if you stumble upon
 bad design or rendering problems (with screen shot if possible), or if you need
 more characters, or if you want to compliment me (I love compliments).
 
+Installation
+------------
+
+You can [download the latest version][] and install it by hand. In the `NoLoopK` variant, the looped lowercase `k` is replaced with a straight version. The `LargeLineHeight` variant is especially useful for users of accented capitals. For more info, see the [CHANGELOG](CHANGELOG.md).
```

Comment:
You probably want to link to https://github.com/belluzj/fantasque-sans/releases/latest